gateway/1.4/modules/ROOT/pages/index.adoc

```diff
@@ -39,6 +39,8 @@ FluentBit implementation enables log output to local files, or to aggregators su
 == Replicas
 A runtime unit is called a `replica`. While a single runtime can support multiple backend APIs for high availability, it is best to deploy Flex Gateway as a cluster with multiple replicas running in parallel. Using clusters increases gateway performance and robustness, and is recommended for high-performance applications that must scale.
 
+A single Flex Gateway can support multiple backend APIs. To support more backend APIs, you can deploy multiple replicas or additional Flex Gateways. Increase replica CPU and memory for higher throughput and increased performance.
```

Comment:
```suggestion
A single Flex Gateway can support multiple backend APIs. To support the most amount of backend APIs, you can deploy multiple replicas or additional Flex Gateways. Increase replica CPU and memory for higher throughput and increased performance.
```
Maybe something like this? Saying "more" is just going to inspire the customer to ask, "More than what?"

Reply:
But maybe it's equally confusing to say "the most amount of", because then they'll ask what that amount is.

So, yeah, maybe it's all good.